sentences/sv/_common.yaml

```diff
@@ -50,11 +50,16 @@ expansion_rules:
   name: "{name}"
   area: "{area}"
   what_is: "(vad är | vad är det | vad är det för | vilken)"
-  is_it: "(är det)"
+  is_it: "(är det | är)"
   brightness: "{brightness} [percent]"
   turn_on: "( tänd | sätt på | slå på | starta )"
   turn_off: "( släck | stäng av | stoppa )"
+  in_on: "(i | på)"
   temp: "(temperatur)"
+  all: "(all | alla )"
+  lightning: "(ljuset | belysning[en] | lampan | lampor[na])"
+  hot_cold: "(varmt | varm | kallt | kall | het | svalt )"
```

Comment:
expansion rules should be in Swedish, not English. Unless they wrap a slot name like `{name}`

Reply:
ok fixed in additional commit